tests/testthat/test-slice.R

```diff
@@ -431,6 +430,10 @@ test_that("slice has informative error messages", {
     "# Can't index beyond the end of a vector"
     vec_slice(1:2, 3L)
     vec_slice(1:2, -3L)
+
+    "# Can't be matrix"
```

Comment:
I don't see the corresponding `verify_errors()`. The title here should be the same title as the `test_that()` block where the verify-errors is located.